string serial                         # Tracker serial number
---
deepdive_ros/Tracker[] trackers           # Motor calibration info
